README.md

```diff
@@ -149,21 +156,27 @@ toybox = { "doll" => 'Sally' }
 
 Here is a good editor for regexp, maybe try and do it without using it first tho: https://rubular.com/
 
+I wasn't able to do this without the cheat sheet from that editor but this is my cheat assisted answer:
+
+`^\.(ts|tsx)(\.erb)?$`
+
+The problem was the first ? which allowed zero matches of (ts|tsx) (matching ., ..erb, .erb)
+
 #### Given the following file locations, what are the outputs?
 
 ```ruby
 # /User/gollum/precious/ring.rb
 File.dirname(__FILE__)
 ```
 
-- *give me answer*
+- 'precious'
 
 ```ruby
 # /User/tom/junk/ring.rb
 File.path(__dir__)
 ```
 
-- *give me answer*
+- '/User/tom/junk/'
```

Comment:
`__dir__` is a method on `Kernal` and will return the absolute file path. This was kind of a trick question because `File.path` is redundant here and will return the same as `__dir__`: `/User/tom/junk/ring.rb`

https://ruby-doc.org/core-2.5.1/Kernel.html#method-i-__dir__